.mk/k8s.mk

```diff
@@ -201,111 +202,46 @@ k8s-nsmgr-save:  $(addsuffix -save,$(addprefix ${CONTAINER_BUILD_PREFIX}-,$(NSMG
 .PHONY: k8s-nsmgr-load-images
 k8s-nsmgr-load-images:  k8s-start $(addsuffix -load-images,$(addprefix ${CLUSTER_RULES_PREFIX}-,$(NSMGR_CONTAINERS)))
 
+
 VPPAGENT_DATAPLANE_CONTAINERS = vppagent-dataplane vppagent-dataplane-dev
+
 .PHONY: k8s-vppagent-dataplane-build
 k8s-vppagent-dataplane-build:  $(addsuffix -build,$(addprefix ${CONTAINER_BUILD_PREFIX}-,$(VPPAGENT_DATAPLANE_CONTAINERS)))
+
  .PHONY: k8s-vppagent-dataplane-save
 k8s-vppagent-dataplane-save:  $(addsuffix -save,$(addprefix ${CONTAINER_BUILD_PREFIX}-,$(VPPAGENT_DATAPLANE_CONTAINERS)))
+
  .PHONY: k8s-vppagent-dataplane-load-images
 k8s-vppagent-dataplane-load-images:  k8s-start $(addsuffix -load-images,$(addprefix ${CLUSTER_RULES_PREFIX}-,$(VPPAGENT_DATAPLANE_CONTAINERS)))
 
-.PHONY: k8s-secure-intranet-connectivity-build
-k8s-secure-intranet-connectivity-build:
-
-.PHONY: k8s-secure-intranet-connectivity-save
-k8s-secure-intranet-connectivity-save:
-
-.PHONY: k8s-secure-intranet-connectivity-load-images
-
-k8s-secure-intranet-connectivity-load-images: 
-
-.PHONY: k8s-vppagent-passthrough-nse-build
-k8s-vppagent-passthrough-nse-build:
-
-.PHONY: k8s-vppagent-passthrough-nse-save
-k8s-vppagent-passthrough-nse-save:
 
-.PHONY: k8s-vppagent-passthrough-nse-load-images
-k8s-vppagent-passthrough-nse-load-images:
-.PHONY: k8s-skydive-build
-k8s-skydive-build:
+.PHONY: k8s-icmp-responder-nse-build
+k8s-icmp-responder-nse-build: k8s-test-nse-build
 
-.PHONY: k8s-skydive-save
-k8s-skydive-save: k8s-skydive-build
+.PHONY: k8s-icmp-responder-nse-save
+k8s-icmp-responder-nse-save: k8s-test-nse-save
```

Comment:
These changes do not belong to this patch. Please separate them and explain what is the purpose.
I am also inclined to discuss a more radical change to our Makefile system.

Reply:
Reverted it : 741cbb5